### DISCUSSION TITLE
no output to the gamecube

### DISCUSSION THREAD

**Discussion by 23r0C00l** [darthcloud/BlueRetro#832] [Q&A]
Hey everyone 
I recently setup an esp32 with blueretro (1.9 hw2) on a gamecube (I also have picoboot/swiss installed and running fine) with just data(1234)/3.3v/ground to start and I plan to add leds and power relay later. I flashed with the "BlueRetro_gamecube.bin" file. I have been able to power up and set the system to GC in the ota config. I have successfully pair 2 ps4 controllers, but I can not get any response from the controllers. It is like the esp32 is not emulating any controllers.. what am I doing wrong here? I am not new to electronics, so I'm pulling my hair out here. thanks for any help you can give.

**Comment by darthcloud** [comment]
You can't use HW2 FW without the sense line connected, use HW1

**Reply by 23r0C00l** [reply]
Thanks so much i was going crazy
